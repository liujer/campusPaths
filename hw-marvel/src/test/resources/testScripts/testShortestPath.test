# Tests if findPath can actually find the shortest path rather
# than any path

LoadGraph testShortest testTSV.tsv

FindPath testShortest one four

FindPath testShortest four one