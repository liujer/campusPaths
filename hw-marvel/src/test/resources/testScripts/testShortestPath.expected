# Tests if findPath can actually find the shortest path rather
# than any path

loaded graph testShortest

path from one to four:
one to four via e14

path from four to one:
four to one via e14